src/main/java/mate/academy/springboot/swagger/util/SortParser.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public class SortParser {
```

Comment:
```suggestion
public class SortUtil {
```
it would be better to use such name, also you can add a constructor without parameters with `private` access modifier